README.md

```diff
@@ -1,95 +1,7 @@
-# MLonCode research playground [![PyPI](https://img.shields.io/pypi/v/sourced-ml.svg)](https://pypi.python.org/pypi/sourced-ml) [![Build Status](https://travis-ci.org/src-d/ml.svg)](https://travis-ci.org/src-d/ml) [![Docker Build Status](https://img.shields.io/docker/build/srcd/ml.svg)](https://hub.docker.com/r/srcd/ml) [![codecov](https://codecov.io/github/src-d/ml/coverage.svg)](https://codecov.io/gh/src-d/ml)
+# MLonCode Core Library
+ [![Build Status](https://travis-ci.org/src-d/ml-core.svg)](https://travis-ci.org/src-d/ml-core)
+ [![codecov](https://codecov.io/github/src-d/ml-core/coverage.svg)](https://codecov.io/gh/src-d/ml-core)
+ [![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/ambv/black)
```

Comment:
let's keep relevant sections: Contributions,  License, Glossary.

And keep the link to https://github.com/src-d/awesome-machine-learning-on-source-code
So let's have the first phrase as it was  `This project is the foundation for [MLonCode](https://github.com/src-d/awesome-machine-learning-on-source-code) research and development.`

Reply:
Done.